baseplate/context/cassandra.py

```diff
@@ -3,7 +3,10 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
+import six
```

Comment:
I think this is the first time we've explicitly used `six` in baseplate. Either it should get added into setup.py etc. or we should just add the relevant bits to the `_compat` module which is sorta a mini six. I slightly lean to the latter since it's one fewer thing to worry about.